setup.cfg

```diff
@@ -19,6 +19,7 @@ packages = eppo_client
 python_requires = >=3.6
 include_package_data=True
 install_requires =
-    pydantic<2
+    pydantic
+    pydantic-settings
```

Comment:
why is there a `setup.cfg` and `requirements.txt` file?

Reply:
I can't know what the intention was here, but I believe the convention is about abstract vs concrete dependencies

https://martin-thoma.com/python-requirements/#abstract-dependencies